scipy/sparse/linalg/_isolve/_gcrotmk.py

```diff
@@ -221,10 +221,10 @@ def gcrotmk(A, b, x0=None, *, rtol=1e-5, atol=0., maxiter=1000, M=None, callback
         According to [2]_, good values are around m.
         Default: m
```

Comment:
```suggestion
        Default: `m`
```

Reply:
could you add single backticks around `m` here?

```
        According to [2]_, good values are around `m`.
        Default: `m`
```

Reply:
done!